apmproxy/client.go

```diff
@@ -46,13 +46,15 @@ const (
 	defaultDataForwarderTimeout time.Duration = 3 * time.Second
 	defaultReceiverAddr                       = ":8200"
 	defaultAgentBufferSize      int           = 100
+	defaultMaxBatchSize         int           = 50
```

Comment:
@simitt I am not sure if it would work in every case since I can't think of a direct correlation between buffer size and batch size. Due to this, I believe, giving only buffer size will make the configuration a bit more complicated.